.vscode/settings.json

```diff
@@ -15,5 +15,6 @@
   "coverage-gutters.showRulerCoverage": false,
   "coverage-gutters.highlightdark": "rgb(61, 153, 112, .05)",
   "coverage-gutters.noHighlightDark": "rgb(255, 65, 54, .05)",
-  "coverage-gutters.partialHighlightDark": "rgb(255, 133, 27, .05)"
+  "coverage-gutters.partialHighlightDark": "rgb(255, 133, 27, .05)",
+  "python.pythonPath": "C:\\Users\\jdadams\\AppData\\Local\\Programs\\ArcGIS\\Pro\\bin\\Python\\envs\\reporter\\python.exe"
```

Comment:
this should be unnecessary if you update your vscode. I'm pretty sure the experimental opt in feature has launched.

Reply:
I missed that experiment. Thanks!